plugins/services/src/js/reducers/form/EnvironmentVariables.js

```diff
@@ -0,0 +1,106 @@
+import {
+  ADD_ITEM,
+  REMOVE_ITEM,
+  SET
+} from '../../../../../../src/js/constants/TransactionTypes';
+import Transaction from '../../../../../../src/js/structs/Transaction';
+
+module.exports = {
+  JSONReducer(state, {type, path, value}) {
+    if (this.env == null) {
+      this.env = [];
```

Comment:
Not exactly sure what is going on here, what is this `env` array? Could you maybe add some comments? This is hard to decipher.